packages/palette/src/elements/ProgressBarTimer/ProgressBarTimer.tsx

```diff
@@ -16,7 +16,7 @@ export const ProgressBarTimer: React.SFC<{
   highlight = "purple100",
 }) => {
   const secondsRemaining = DateTime.fromISO(countdownEnd).diff(
-    DateTime.fromISO(currentTime),
+    DateTime.fromISO(currentTime.toString()),
```

Comment:
Why was this change needed?

Reply:
If `currentTime` is passed as a `DateTime` instead of a `string` the `DateTime.fromISO` wont be able to parse it unless it's converted to a string.

Reply:
This seems like a good thing to wrap some tests around.